cms/static/cms/sass/components/_structureboard.scss

```diff
@@ -470,16 +470,15 @@
         right: 0;
```

Comment:
I think the `right: ` in line 447 needs adjustment, too.

Reply:
I'm not sure if this is the button that resizes the modal or not but if it is, I'll need so mirror the JavaScript code too when resizing.

Reply:
No, that's the locked icon for text-enabled plugins in the structure board.

And yes, both the icon and the js code for resizing the modal would need to be adjusted, if you moved the resize button to the right.